src/pages/Home.js

```diff
@@ -16,43 +16,34 @@ export default function Home() {
     localStorage.removeItem(`user`);
     navigate("/login");
   }
-
-  console.log(user);
+  
   return(
-    <> 
+    <div> 
       <h1>home</h1>
       <h5>not ready</h5>
-      <StyledSection>
-        <h3>{user}</h3>
-        <StyledDiv>
-          <p>test</p>
-          <button onClick={()=> {logout();}}>logout</button>
-        </StyledDiv>
-      </StyledSection>
-    </>
+      <StyledNavBar>
+        <Div1>
+          <button onClick={()=> logout()}>{user} logout</button>
+        </Div1>
+      </StyledNavBar>
+    </div>
   );
 }
 
-export const StyledSection = styled.section`
-  border-top: solid;
-  border-width: 1px;
-  display: grid;
-  column-gap: 50px;
-  grid-template-columns: auto auto auto;
-  
+export const Div1 = styled.div `
+border: 0.1 px solid;
+width: 100%;
+display: grid;
+float: left;
+width: 33.33%;
+`
+export const StyledNavBar = styled.section`
+  border-top: 1px solid;
+  display: flex;
+  justify-content: space-between;
   position: fixed;
-  left: 0;
-  bottom: 0;
   width: 100vw;
-  padding: 10px;
-`
-
-export const StyledDiv = styled.div`
-  border: 1px solid;
-  background-color: lightgrey;
-
-  position: absolute;
-  top: -100px;
-  left: 50px;
-  fill: #828a95;
+  height: 45px;
+  bottom: 0;
+  layer: 22;  
```

Comment:
Layer property still doesn't exist. Please remove it or adjust it if you meant something else by it.

Reply:
You are right. The layer are not needed now. It will come later.